packages/photon/package.json

```diff
@@ -37,11 +42,12 @@
 		"seed-random": "^2.2.0"
 	},
 	"devDependencies": {
-		"@automattic/calypso-typescript-config": "workspace:^"
+		"@automattic/calypso-typescript-config": "workspace:^",
+		"@types/seed-random": "^2.2.1"
 	},
 	"scripts": {
-		"clean": "rm -rf dist",
-		"build": "transpile",
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
```

Comment:
`copy-assets` is only needed if the package has style sheets or images, which is not the case.